orte/mca/oob/tcp/help-oob-tcp.txt

```diff
@@ -113,3 +113,27 @@ The connection was rejected.
 Static ports were requested while orte_fwd_mpirun_port was set.
 Both options cannot be simultaneously set. Please either set
 orte_fwd_mpirun_port=false or remove any static port directives.
+#
+[version mismatch]
```

Comment:
Did you mean to add this "Version mismatch" message in this PR?

Reply:
Doesn't do any harm and it wasn't clear if the conflict required it to be there.